programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -10,6 +10,7 @@ pub use self::{
         SyscallGetSysvar,
     },
 };
+
```

Comment:
Can you run `cargo fmt --all`? I think it does not like empty lines in between `use`s